File: Test projects/.NET6.0/WinFormsTestAppNet6Vb/Services/ExampleService.vb
Namespace WinformsTestAppNet6Vb.Services

    Public Interface IExampleService

        Sub DoSomething()

    End Interface

    Public Class ExampleService
        Implements IExampleService

        Public Sub DoSomething() Implements IExampleService.DoSomething

            ' Do something
            MessageBox.Show("Do something")

        End Sub
    End Class

End Namespace

File: Test projects/.NET6.0/WinFormsTestAppNet6CSharp/Services/ExampleService.cs
namespace WinFormsTestAppNet6CSharp.Services
{
    public interface IExampleService
    {
        void DoSomething();
    }

    public class ExampleService : IExampleService
    {
        public void DoSomething()
        {
            // Do something
            MessageBox.Show("Do Something");
        }
    }
}
